docs/DeveloperGuide.md

```diff
@@ -330,14 +393,49 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * 2a. The list is empty.
 
   Use case ends.
+* 3a. The given format is invalid.
+    * 3a1. System shows an error message.
 
-* 3a. The given index is invalid.
+      Use case resumes at step 2.
+* 3b. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3b1. System shows an error message.
 
       Use case resumes at step 2.
+* *a. Connection lost.
+    * *a1. System informs the user.
 
-*{More to be added}*
+      Use case ends.
+
+**Use case: Find a person**
+
+**MSS**
+
+1.  User requests to list persons
+2.  System shows a list of persons
+3.  User requests to find a specific person in the list
+4.  System shows the person
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+* 3a. The given format is invalid.
```

Comment:
I think same with the above, I think use case should not resume from step 2, but it should prompt the user for the correct data.

```suggestion
* 3a. The given format is invalid.
    * 3a1. TutorTrack shows an error message.
    * 3a2. TutorTrack requests for the correct data
    * 3a3. Tutor enters new data
  
    Steps 3a2-3a3 are repeated until the data entered are correct.
  
    Use case resumes from step 4.
```